src/schema.js

```diff
@@ -52,63 +63,89 @@ const blocks = {
     },
     files: {
         parents: ['server'],
+        description: "Configures file serving.",
+        examples: ['simple-fileserving', 'spa-serving'],
         labelled: false
     },
     jwt: {
         parents: ['definitions'],
+        description: "Configures a JSON Web Token access control.",
+        examples: ['jwt-access-control', 'creating-jwt'],
         labelled: true
     },
     jwt_signing_profile: {
         parents: ['definitions'],
+        description: "Configure a JSON Web Token signing profile which is referenced in the `jwt_sign()` function.",
+        examples: ['creating-jwt'],
         labelled: true
     },
     oauth2: {
```

Comment:
The id for this block in REFERENCE.md is `oauth2-cc-block`, while this links to `oauth2-block`.

Reply:
See above.